CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 1.70.0
+* Fix: flaky TestMetricsDisabled unit test. #3588
```

Comment:
dont need to put unit test fixes in changelog, this doesnt effect any external functionality